tools/accuracy_checker/accuracy_checker/postprocessor/letterbox_postrocessing_resize.py

```diff
@@ -0,0 +1,60 @@
+"""
+Copyright (c) 2021 Intel Corporation
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from ..representation import DetectionPrediction, DetectionAnnotation
+from ..postprocessor.postprocessor import Postprocessor
+
+
+class LetterboxPostprocessingResize(Postprocessor):
+    __provider__ = 'letterbox_postprocessing_resize'
```

Comment:
what the difference from frcnn resize and can they be combined in one?

Reply:
Probably